packages/web3torrent/scripts/start.js

```diff
@@ -61,4 +61,18 @@ void (async () => {
   devServer.on('close', code => {
     process.exit(code);
   });
+
+  const trackerServer = spawn(cmd, ['run', 'start:tracker']);
+
+  trackerServer.stdout.on('data', data => {
+    console.log(data.toString());
+  });
+
+  trackerServer.stderr.on('data', data => {
+    throw data.toString();
+  });
+
+  trackerServer.on('close', code => {
+    process.exit(code);
+  });
```

Comment:
I am not sure this server is being torn down correctly. If I run `yarn start`, kill the process and then run `yarn start` again, I get this:

```
➜  web3torrent git:(sp/w3t-tracker) ✗ yarn start 
yarn run v1.19.1
$ node scripts/start.js
The USE_GANACHE_DEPLOYMENT_CACHE option is set. Using ganache in shared mode with cached deployments. Port = 8547.
Found shared ganache instance running on http://localhost:8547.
Using the deployments cache at /Users/georgeknee/statechannels/monorepo/.ganache-deployments/ganache-deployments-8547.json.
Contract SingleAssetPayments already exists address: 0x430869383d611bBB1ce7Ca207024E7901bC26b40
$ node tracker/index.js

$ /Users/georgeknee/statechannels/monorepo/node_modules/.bin/react-scripts start


/Users/georgeknee/statechannels/monorepo/packages/web3torrent/scripts/start.js:72
    throw data.toString();
    ^
listen EADDRINUSE: address already in use :::8000

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
➜  web3torrent git:(sp/w3t-tracker) ✗ 
```

Reply:
I.... can't seem to get that error. I ran it like 20 times, hitting `CTRL` + `C` at different moments, and it always restarts okay.
Are you sure that you are not running the tracker in another tab or something?

Reply:
Ok I killed all my processes that listen on `8000` and now the problem seems to go away 🤷‍♂

Reply:
I gave it another try, trying to show the debug messages, and everything breaks.
I pushed another commit. Now if you run `DEBUG="web3torrent:tracker" yarn start` it should show you all the logs of the tracker in the console

Reply:
Thanks! I think I had tried this once maybe that's why I had issues.